### PR TITLE
Feat(root): Apply code format on commit

### PR DESCRIPTION
**Pull request by lqt93**
#115

### CODE REVIEW

**Review by tuhuynh27 — APPROVED** [review]
LGTM

File: .husky/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+_
```
Comment (thien-do) [inline .husky/.gitignore:1]: @tuhuynh27 @lqt93 sorry, just a small question, what does this mean?

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,4 @@
 node_modules
 dist
 .DS_Store
-.idea
+.idea
```
Comment (thien-do) [inline .gitignore:4]: This is not a big deal but usually I'd prefer new line at end file

File: package.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+	"name": "@moai/root",
+	"lint-staged": {
+		"*.{ts,tsx,js,css,json}": "prettier --write"
+	},
+	"devDependencies": {
+		"husky": "^6.0.0",
+		"lint-staged": "^10.5.4",
+		"prettier": "^2.2.1"
+	}
```
Comment (thien-do) [inline package.json:10]: @lqt93 I'm sorry but can you use exact version? It's just I want to manually control the dependencies upgrading

**Review by thien-do — CHANGES_REQUESTED** [review]
Great PR @lqt93 ! Just some small questions

File: .gitignore (hunk 2 above)
Comment (tuhuynh27) [reply .gitignore:4]: Oh sorry, please help me to add a new line at end file @lqt93

**Comment by thien-do** [comment]
Oh looks like we can merge https://github.com/moaijs/moai/pull/135/files which includes this! Great work @lqt93 !

**Pull request closed without merging**